shub/settings/main.py

```diff
@@ -128,7 +128,29 @@
 GRAVATAR_DEFAULT_IMAGE = "retro"
 # An image url or one of the following: 'mm', 'identicon', 'monsterid', 'wavatar', 'retro'. Defaults to 'mm'
 
+
+# Default Django logging is WARNINGS+ to console
+# so visible via docker-compose logs uwsgi
+LOGGING = {
+    'version': 1,
```

Comment:
tiny detail - let's be overly organized and put this in another settings file called `logging.py` so the developer finds it immediately!

Reply:
Actually if I merge from a push (after @victorsndvg and I test) I can do this easily.

Reply:
Ahh ... I already ... see below :-)